### DISCUSSION TITLE
How to space out the headlabel and taillabel for MultiDiGraph ?

### DISCUSSION THREAD

**Discussion by nyue** [networkx/networkx#4913] [Q&A]
```
import networkx as nx

G = nx.MultiDiGraph()
G.add_edge(1,2, color='red',headlabel='abc',taillabel='xyz',labelangle=-180)
G.add_edge(1,2, color='green',headlabel='angle',taillabel='matrix',labelangle=-180)

A = nx.nx_agraph.to_agraph(G)
A.layout(prog='dot')
A.draw('digraph.png')
```

In the resulting image, the text `abc`, `xyz`, `angle` and `matrix` are all squished together, is there a way to convey to dot to allow space so that the labels are more readable?

Or is there a way to layout the graph so that the edges flow horizontally so that the labels have more spaces ?

![digraph](https://user-images.githubusercontent.com/486172/122684952-75c80900-d1bd-11eb-8de2-dc9d619f84fd.png)

Cheers

**Comment by dschult** [comment]
That is really more of a question about `dot` and GraphViz more generally. If someone here knows, then great! But you might try looking/asking the GraphViz experts.

**Reply by nyue** [reply]
Thanks @dschult for the suggestion, googling graphviz helped. I found something I can use for now.
```
G = nx.MultiDiGraph(rankdir="LR")
```
